### PR TITLE
Jf/dev/remove receivers and senders

### PR DESCRIPTION
**Pull request by joaofigueiredobisect**

### CODE REVIEW

File: cpp/libs/ossrf_nmos_api/lib/src/context/resource_map.cpp [hunk 1]
```diff
@@ -68,3 +68,39 @@ expected<nmos_resource_ptr> resource_map_t::find_resource(const std::string& res
...
+        {
+            if(resource->get_resource_type() == nmos::types::receiver)
+            {
+                ids.push_back(id);
+            }
+        }
+    }
+
+    return ids;
+}
```
Comment (pedro-alves-ferreira) [inline cpp/libs/ossrf_nmos_api/lib/src/context/resource_map.cpp:106]: Missing line here. Doesn't clang-format get this?

File: cpp/libs/ossrf_nmos_api/lib/src/context/resource_map.h [hunk 2]
```diff
@@ -34,6 +34,9 @@ namespace ossrf
         void erase(std::string);
 
         bisect::expected<nmos_resource_ptr> find_resource(const std::string& resource_id);
+        std::vector<std::string> get_sender_ids() const;
+        std::vector<std::string> get_receiver_ids() const;
+
```
Comment (pedro-alves-ferreira) [inline cpp/libs/ossrf_nmos_api/lib/src/context/resource_map.h:39]: Extra line.

**Review by pedro-alves-ferreira — CHANGES_REQUESTED** [review]

File: cpp/libs/ossrf_nmos_api/lib/src/context/resource_map.cpp (hunk 1 above)
Comment (joaofigueiredobisect) [reply cpp/libs/ossrf_nmos_api/lib/src/context/resource_map.cpp]: Clang-format doesn't get this, but the line is now added

**Review by massis08 — APPROVED** [review]
LGTM

Just rebase main in this PR when PR #4 gets merged

**Pull request merged**